css/at-rules/supports.json

```diff
@@ -109,12 +109,10 @@
                 "notes": "See <a href='https://crbug.com/979041'>bug 979041</a>."
               },
               "safari": {
-                "version_added": false,
-                "notes": "See <a href='https://webkit.org/b/199237'>bug 199237</a>"
+                "version_added": "14.1"
               },
               "safari_ios": {
-                "version_added": false,
-                "notes": "See <a href='https://webkit.org/b/199237'>bug 199237</a>"
+                "version_added": "14.1"
```

Comment:
This should be 14.5. 

Safari 14.1 looks correct on macOS, but the iOS version is wrong. 

This change would have been previewed available in Safari Technology Preview 114 and shipped with Safari 14.1 on macOS and Safari on iOS 14.5.

Reply:
In a situation like (when I'm very sure of the suggested change) this I usually apply the change myself rather than waiting for the contributor to do it.

Reply:
I'll go ahead and make the change and merge.

Reply:
@foolip Sorry for the incomplete pull request and thank you for the fix.

Reply:
No problem, it was a simple fix!